src/LIKWID.jl

```diff
@@ -75,10 +75,26 @@ import .GPUMarker: gpumarker, @gpumarker
 export GPUMarker, gpumarker, @gpumarker
 include("frequency.jl")
 
+function perf_event_paranoid()
+    open("/proc/sys/kernel/perf_event_paranoid", "r") do io
```

Comment:
Does `/proc/sys/kernel/perf_event_paranoid` always exist and is it always readable by users? Is assume the answers are yes and yes. Just wondering if we need a default fallback here.

Reply:
Always on Linux yeah, let me throw in an ispath